tests/ci/files/rhel-8-3-1.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
do we need this file here?

Reply:
same applies to `rhel-8-4-0.json`

Reply:
True. I removed yesterday the RHEL support and I forgot those files. Thanks!